src/yogeshNagar/Test1_Program1_NonRepeatingCharacterString.java

```diff
@@ -0,0 +1,31 @@
+package yogeshNagar;
+
+public class Test1_Program1_NonRepeatingCharacterString {
+
+	int getRepeatingCharCount(String name,char ch) {
+		int count=0;
+		for(int index=0; index<name.length(); index++) {
+			if(name.charAt(index)==ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void passValue(String name) {
+		for(int index=0;index<name.length();index++) {
+			int count = getRepeatingCharCount(name,name.charAt(index));
+			if(count==1) {
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop